cli/src/main.rs

```diff
@@ -87,7 +87,11 @@ enum Command {
     },
     /// Builds the smart contract.
     #[structopt(name = "build")]
-    Build {},
+    Build {
+        /// Generate the contract abi artifacts
+        #[structopt(long = "abi")]
+        generate_abi: bool,
+    },
```

Comment:
This is the output I get for `--help`:
```
$ cargo contract build --help
…
USAGE:
    cargo contract build

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information
```
Could you make the `--abi` flag show up here?